commit a72040a9d929c17dcfb874671b21e4410df4c8c0
Author: Boris Brezillon <bbrezillon@kernel.org>
Date:   Thu Jan 24 15:46:54 2019 +0100

    mtd: spinand: Fix the error/cleanup path in spinand_init()
    
    commit c3c7dbf4887ab3ed9d611cd1f6e16937f8700743 upstream.
    
    The manufacturer specific initialization has already been done when
    block unlocking takes place, and if anything goes wrong during this
    procedure we should call spinand_manufacturer_cleanup().
    
    Fixes: 7529df465248 ("mtd: nand: Add core infrastructure to support SPI NANDs")
    Cc: <stable@vger.kernel.org>
    Signed-off-by: Boris Brezillon <bbrezillon@kernel.org>
    Acked-by: Miquel Raynal <miquel.raynal@bootlin.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/mtd/nand/spi/core.c b/drivers/mtd/nand/spi/core.c
index a78621c7b11d..8c7bf91ce4e1 100644
--- a/drivers/mtd/nand/spi/core.c
+++ b/drivers/mtd/nand/spi/core.c
@@ -1014,11 +1014,11 @@ static int spinand_init(struct spinand_device *spinand)
 	for (i = 0; i < nand->memorg.ntargets; i++) {
 		ret = spinand_select_target(spinand, i);
 		if (ret)
-			goto err_free_bufs;
+			goto err_manuf_cleanup;
 
 		ret = spinand_lock_block(spinand, BL_ALL_UNLOCKED);
 		if (ret)
-			goto err_free_bufs;
+			goto err_manuf_cleanup;
 	}
 
 	ret = nanddev_init(nand, &spinand_ops, THIS_MODULE);
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1310_a72040a9d929c17dcfb874671b21e4410df4c8c0.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1310_a72040a9d929c17dcfb874671b21e4410df4c8c0.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
